pavelib/paver_tests/test_paver_bok_choy_cmds.py

```diff
@@ -37,12 +39,14 @@ def _expected_command(self, name, store=None):
             shard_str='/shard_' + self.shard if self.shard else '',
             exp_text=name,
             a11y_custom_file='node_modules/edx-custom-a11y-rules/lib/custom_a11y_rules.js',
+            verify_xss=verify_xss
         )
         return expected_statement
 
     def setUp(self):
         super(TestPaverBokChoyCmd, self).setUp()
         self.shard = os.environ.get('SHARD')
+        self.env_xss = EnvironmentVarGuard()
```

Comment:
I'd just put it in the test_verify_xss_env_var method instead of setup, since it is only used in one place.